lib/jsdom/living/nodes/RadioNodeList-impl.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+const NodeListImpl = require("./NodeList-impl").implementation;
+
+class RadioNodeListImpl extends NodeListImpl {
+  get value() {
+    this._update();
+    const element = this._list.find(e => e._localName === "input" && e.type === "radio" && e.checked);
```

Comment:
`e.checked` doesn't necessarily represent the checkedness that the spec asks about. As you can see from https://html.spec.whatwg.org/#attr-input-checked it's a pretty complicated relationship. Instead, check `e._checkedness`.